app/src/main/java/com/example/marvelapp/activity/CharactersActivity.kt

```diff
@@ -42,7 +41,7 @@ class CharactersActivity : AppCompatActivity(), KoinComponent, CharacterListener
         binding.charactersActivityLoader.visibility = View.VISIBLE
     }
 
-    private fun displayCharacters(charactersList: List<Character>) {
+    private fun displayCharacters(charactersList: List<com.example.domain.entity.Character>) {
```

Comment:
add the correct import instead of writing the path here

Reply:
done